### PR TITLE
[API-1348] Regex expr. updated - parse api version in swagger urls is…

### PR DESCRIPTION
**Pull request by vladimir-trifonov**
…sue fixed

https://jira.appcelerator.org/browse/API-1348

### CODE REVIEW

File: lib/utility/utils.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function parseModelName(path) {
 	}
 	segments = path.split('/');
 	// Often the first path segment may be an API version number
-	if (segments[0].match(/v?\d+([\.\d+])*/)) {
+	if (segments.length > 1 && segments[0].match(/^v?(\*|\d+([\.\d+])*)/)) {
```
Comment (janvennemann) [inline lib/utility/utils.js]: Could you explain what the `\*` part in he regex is for? That would match `v*`?

**Comment by vladimir-trifonov** [comment]
About:

In lib/utility/utils.js:

> @@ -19,7 +19,7 @@ function parseModelName(path) {
>  	}
>  	segments = path.split('/');
>  	// Often the first path segment may be an API version number
> -	if (segments[0].match(/v?\d+([\.\d+])*/)) {
> +	if (segments.length > 1 && segments[0].match(/^v?(\*|\d+([\.\d+])*)/)) {
>
> Could you explain what the \* part in he regex is for? That would match v*?

It was definitely a mistake. Now it's fixed :)

**Comment by janvennemann** [comment]
Thanks, approved! Will merge and bump the version together with another PR for this connector i'm currently preparing.

**Comment by janvennemann** [comment]
@vladimir-trifonov, can you please bump the version?

**Comment by vladimir-trifonov** [comment]
Done!

**Pull request merged**